com.w1hkj.flrig.metainfo.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 hamradio lu9dce -->
+<component type="desktop-application">
+ <id>com.w1hkj.flrig</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>GPL-3.0</project_license>
+ <name>flrig</name>
```

Comment:
Flrig

(uppercase first letter)

Reply:
@AsciiWolf 4 o 7?

Reply:
?

Reply:
Just change the name to **F**lrig.

Reply:
Looks like it's lowercase also in documentation so let's stick to it.

Reply:
It's both lowercase and uppercase. However, it's uppercase in both Fedora and Ubuntu AppData. Flrig is the same.